playbooks/mysql_init.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+# This play should run locally and target a remote MySQL instance by setting
+# mysql_login_user, mysql_login_password, mysql_login_host, and
+# mysql_login_port
+- name: Configure mysql DBs and users
+  hosts: all
+  sudo: False
+  gather_facts: False
+  vars_files:
+    - roles/analytics_api/defaults/main.yml
+    - roles/analytics_pipeline/defaults/main.yml
+    - roles/common_vars/defaults/main.yml
+    - roles/ecommerce/defaults/main.yml
+    - roles/edxapp/defaults/main.yml
+    - roles/edx_notes_api/defaults/main.yml
+    - roles/hive/defaults/main.yml
+    - roles/insights/defaults/main.yml
+    - roles/programs/defaults/main.yml
+    - roles/xqueue/defaults/main.yml
```

Comment:
Yes, defaults should be overridden for any of these services that are actually being used.